raven_aiohttp.py

```diff
@@ -19,30 +19,38 @@ class AioHttpTransport(AsyncTransport, HTTPTransport):
     def __init__(self, parsed_url, *, verify_ssl=True, resolve=True,
                  timeout=defaults.TIMEOUT,
                  keepalive=True, family=socket.AF_INET, loop=None):
+        self.verify_ssl = verify_ssl
+        self.resolve = resolve
```

Comment:
Yes, but perhaps it would be strange if some variables start with underscore and others don't (`resolve` doesn't contain underscore in `HTTPTransport`) and they belong to one group?